vnext/codegen/react/components/rnwcore/ComponentDescriptors.h

```diff
@@ -0,0 +1,30 @@
+
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
```

Comment:
is this right?

Reply:
This file is codegen'd.  Given that this same codegen is going to be used for external components, I'll look at creating a PR in core to adjust this comment.
I agree it doesn't make sense to generate this: 
https://github.com/facebook/react-native/pull/31516

Reply:
thanks. Separately, if this file is codegen'd, shouldn't be named .g.h?

Reply:
The name of the file is similarly picked by the facebook codegen.  
https://github.com/facebook/react-native/blob/b2dbde36b8f9cb69b1a1f9061c7f19c895d0c317/packages/react-native-codegen/src/generators/components/GenerateComponentDescriptorH.js#L52
Maybe we should look into seeing if thats something they'd be interested in changing.